package/Dockerfile

```diff
@@ -12,10 +12,16 @@ ENV KUSTOMIZE_URL=https://github.com/kubernetes-sigs/kustomize/releases/download
 RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${ARCH}/kubectl && \
     chmod +x kubectl
 RUN if [ "${ARCH}" = "amd64" ]; then ARCH=x86_64; fi && \
-    curl -sfL https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9s_Linux_${ARCH}.tar.gz | tar xvzf -
+    if [ "${ARCH}" != "s390x" ]; then \
+        curl -sfL https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9s_Linux_${ARCH}.tar.gz | tar xvzf -; \
+    else \
+        curl -OL https://github.com/Anarkis/k9s/releases/download/v0.24.15-s390x/k9s; \
```

Comment:
i think we shouldnt commit this until we see if k9s takes the PR from raul to add support for it. should be trivial for that project to build the binaries for us and we dont have to rely on a pinned fork

https://github.com/derailed/k9s/pull/1301

Reply:
Agreed... how much time do you think we should wait for ?

Reply:
PR has been merged and k9s is now available on s390x :smile: 
https://github.com/derailed/k9s/releases/tag/v0.25.3

Reply:
Done. Updated the PR